packages/@uppy/google-drive/src/GoogleDrive.jsx

```diff
@@ -0,0 +1,86 @@
+import { UIPlugin } from '@uppy/core'
+import { Provider } from '@uppy/companion-client'
+import { h } from 'preact'
+
+import packageJson from '../package.json'
+import DriveProviderViews from './DriveProviderViews.js'
+import locale from './locale.js'
+
+export default class GoogleDrive extends UIPlugin {
+  static VERSION = packageJson.version
+
+  constructor (uppy, opts) {
+    super(uppy, opts)
+    this.id = this.opts.id || 'GoogleDrive'
+    this.title = this.opts.title || 'Google Drive'
+    Provider.initPlugin(this, opts)
+    this.title = this.opts.title || 'Google Drive'
+    this.icon = () => (
```

Comment:
It's unrelated to this PR but I've always felt we should have a SVG loader and not pollute the class with this. Reminded of this as the PRs come by.